l10n_nl_hr_expense/data/mail_alias_data.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="hr_expense.mail_alias_expense" model="mail.alias">
+        <field name="alias_name">uitgaven</field>
```

Comment:
Should this not be English with an entry in the nl po file?

Reply:
This is not a translatable field.